enclave/src/openfinex/openfinex_client.rs

```diff
@@ -145,7 +147,7 @@ impl TcpClient {
     /// We're ready to do a read.
     fn do_read(&mut self) -> io::Result<usize> {
         //FIXME: maybe we can use up buffer read here?
-        let mut buffer = [0 as u8; 512]; // Dummy buffer. will not be necessary with tls client
+        let mut buffer = [0 as u8; 1028]; // Dummy buffer. will not be necessary with tls client
```

Comment:
Is this intentional, `1028`? A power of 2 would be `1024`?

Reply:
Very much intentional, yes. The 4 bytes extra are for the ws header, which gives 1024 bytes for the payload. Not sure if that's smart though.

Reply:
ah okay thanks, good to know